ert3/algorithms/_sensitivity.py

```diff
@@ -118,7 +119,7 @@ def fast_sample(
 
 def fast_analyze(
     parameters: MutableMapping[str, Distribution],
-    model_output: RecordCollectionMap,
+    model_output: Dict[int, Dict[str, RecordTransmitter]],
```

Comment:
Just wandering whether we shouldn't use `Mapping` instead of `Dict`; where `Mapping` implies immutability

Reply:
Not sure what the advantage of using Mapping is

Reply:
Not explicit, but it describes the parameter being immutable.